main.java

```diff
@@ -1 +1,50 @@
+import java.util.*;
+class stdmarks{
 
+              public static void main (String args[]){
+              
+              
+                  System.out.println("enter the name of the student");        
```

Comment:
You can do the prompt text as "... student: ". Note the ": ", just adds the aesthetics of the user prompt.